Docs/21726 (#31627)
hashicorp/terraform-provider-aws: website/docs/r/wafv2_web_acl_logging_configuration.html.markdown

If you are capturing logs for Amazon CloudFront, always create the firehose in US East (N. Virginia).
Be sure to give the data firehose, cloudwatch log group, and/or s3 bucket a name that starts with the prefix `aws-waf-logs-`.

-> **Note:** When logging from a WAFv2 Web ACL to a CloudWatch Log Group the WAFv2 service attempts to create/update a

> generic Log resource policy with a name `AWSWAF-LOGS`, if there are a large number of Web ACLs or the account frequently

Reviewer: Breaking this sentence into two separate sentences, adjusting some capitalization.

```suggestion
generic Log Resource Policy with a name `AWSWAF-LOGS`. If there are a large number of Web ACLs, or the account frequently
```

> creates and destroys Web ACLs this policy will hit the max policy size and this resource type will fail to be

Reviewer: Adding a comma to break up the sentence.

```suggestion
creates and destroys Web ACLs, this policy will hit the max policy size and this resource type will fail to be
```

> created(Full details in this [issue](https://github.com/hashicorp/terraform-provider-aws/issues/25296)). To avoid this

Reviewer: Minor restructuring of the parenthesis phrasing here.

```suggestion
created (more details can be found in [this issue](https://github.com/hashicorp/terraform-provider-aws/issues/25296)). To avoid this
```

> happening a specific resource policy can be manged see CloudWatch Log Group example below.

Reviewer: Corrected typo and added hyperlink to example

```suggestion
happening, a specific resource policy can be managed. See [CloudWatch Log Group](#with-cloudwatch-log-group-and-managed-cloudwatch-log-resource-policy) example below.
```

}
```

> ## With Cloudwatch Log group and managed CloudWatch Log Resource Policy (See Notes)

Reviewer: - Removed the `(See Notes)` so that we can hyperlink to this example. 
- Corrected title level to match other examples. 
- Adjusted capitalization in a couple of places.

```suggestion
### With CloudWatch Log Group and managed CloudWatch Log Resource Policy
```
